### ISSUE TITLE
Remove deprecated lifecycle methods

### ISSUE THREAD

**Issue by dstroot** [Wufe/react-particles-js#63]
Component does not pass react strict mode.  Need to remove "componentWillMount" and "componentWillUpdate".  Since they don't seem to actually do anything it should be fine.

Cheers!

```
index.js:1452 Warning: Unsafe lifecycle methods were found within a strict-mode tree:

componentWillMount: Please update the following components to use componentDidMount instead: SideEffect(NullComponent), t

componentWillUpdate: Please update the following components to use componentDidUpdate instead: t

Learn more about this warning here:
https://fb.me/react-strict-mode-warnings
```

**Comment by Wufe** [comment]
Thanks for your tip. The change will be included in next release (v2.4.0)

**Comment by dstroot** [comment]
Cool!
